benchmarks/benchmarks/utils.py

```diff
@@ -37,11 +37,61 @@ def get_graph(name):
         print(name + " doesn't exist")
         return None
 
+class ogb_data(object):
+    def __init__(self, g, num_labels):
+        self._g = g
+        self._num_labels = num_labels
+
+    @property
+    def num_labels(self):
+        return self._num_labels
+
+    @property
+    def num_classes(self):
+        return self._num_labels
+
+    def __getitem__(self, idx):
+        return self._g
+
+def load_ogb_product(name):
+    from ogb.nodeproppred import DglNodePropPredDataset
+
+    os.symlink('/tmp/dataset/', os.path.join(os.getcwd(), 'dataset'))
```

Comment:
Change it to an environment variable or something like `~/.dgl/ogb/`?

Reply:
You can make the decision and let me know:)